clients/java/src/main/java/marquez/client/models/Run.java

```diff
@@ -45,14 +46,16 @@ public Run(
       @Nullable final Instant startedAt,
       @Nullable final Instant endedAt,
       @Nullable final Long durationMs,
-      @Nullable final Map<String, String> args) {
+      @Nullable final Map<String, String> args,
+      @Nullable final Object facets) {
```

Comment:
Why are the facets all `Object` in the client?

Reply:
I think it'd make sense to use `Map<String, Object>` instead. I'll make the change 👍